Sprint 1

Neagz opened the pull request:
Pull Request Sprint_1

Neagz committed:
Верстка проекта

Neagz committed:
Netlify config

Neagz committed:
Editing the README.md

Neagz committed:
Editing the README.md and package.json (npm run start)

Neagz committed:
Correcting a New Line error

Neagz committed:
Correcting a New Line error

Neagz committed:
Correcting a New Line error

Neagz committed:
Correcting a New Line error

Neagz committed:
Correcting a New Line error

Neagz committed:
Correcting version node.js

Neagz committed:
Корректировка согласно чек-листу для самопроверки
- Добавлены ссылки на страницы в READMI.md
- Добавлена ссылка на макет Figma
- Корректировка наименований id и title для страниц: "Профиль"/"Изменить профиль"/"Изменить пароль"

mikebars1995 commented on src/pages/navigate/navigate.hbs:
```diff
@@ -0,0 +1,10 @@
+<ul>
+    <li>{{> Link page="login" position="center" style="profile_primary" name="Вход"}}</li>
+    <li>{{> Link page="registration" position="center" style="profile_primary" name="Регистрация"}}</li>
+    <li>{{> Link page="list" position="center" style="profile_primary" name="Чат [Заглушка]"}}</li>
+    <li>{{> Link page="profile" position="center" style="profile_primary" name="Профиль"}}</li>
+    <li>{{> Link page="profile_edit" position="center" style="profile_primary" name="Изменить профиль"}}</li>
+    <li>{{> Link page="profile_password" position="center" style="profile_primary" name="Изменить пароль"}}</li>
+    <li>{{> Link page="500" position="center" style="profile_primary" name="Ошибка 500"}}</li>
+    <li>{{> Link page="404" position="center" style="profile_primary" name="Ошибка 404"}}</li>
+</ul>
```
**Нужно исправить**

Нужно обернуть список с ссылками в тег `nav` по семантике, так как это навигация по сайту.

Neagz replied:
Исправил

mikebars1995 commented on src/style.css:
```diff
@@ -0,0 +1,261 @@
+  width: 100%;
+  padding-left: 30px;
+  padding-right: 30px;
+  background: #ffffff;
```
**Можно лучше**

все **цвета** лучше вынести в `css`-переменные и использовать их, чтобы можно было мгновенно понимать, что это за цвет, и можно было бы подкорректировать его в одном месте  и не искать дублирования во всем коде

Neagz replied:
Исправил

mikebars1995 commented on src/components/input/input.hbs:
```diff
@@ -0,0 +1,14 @@
+<div class="form-item">
+    <div class="form-item__group">
+        <label class="form-item__label" for="{{id}}">{{label}}</label>
+        <input class="form-item__input" id="{{id}}" type="{{type}}" value="{{value}}"
```
**Нужно исправить**

По заданию 1го спринта нужно давать инпутам **конкретные** атрибуты `name`, чтобы в 3м спринте не менять ничего при подключении к `Api`. Скрин из задания https://disk.yandex.ru/i/lSPqeGrAa20ayQ

Neagz replied:
Исправил

mikebars1995 reviewed (commented):
Здравствуйте. (**Нужно развернуть общий комментарий ↓**)

Посмотрите на *гитхабе* все комментарии к коду (нужно прокрутить вниз страницу там)

Работа проделана огромная


Что сделано отлично:


* `Readme` хорошо оформлен
* Проект задеплоен и отлично работает
* Отлично, что не забыли про `.gitignore`
* Отлично, что ограничиваете версию `node`
* Отлично, что нет `EOF` ошибок в гите
* Хорошая структура папок и файлов
* Отлично, что не забываете про тег `main` для главной части страницы

Что нужно исправить:

* Нужно обернуть список с ссылками в тег `nav` по семантике, так как это навигация по сайту.
* По заданию 1го спринта нужно давать инпутам **конкретные** атрибуты `name`, чтобы в 3м спринте не менять ничего при подключении к `Api`. Скрин из задания https://disk.yandex.ru/i/lSPqeGrAa20ayQ 

Что можно улучшить:

* все **цвета** лучше вынести в `css`-переменные и использовать их, чтобы можно было мгновенно понимать, что это за цвет, и можно было бы подкорректировать его в одном месте  и не искать дублирования во всем коде

Исправьте, пожалуйста, недочеты и работа будет принята. Пожалуйста, проверьте работоспособность проекта и наличие возможных ошибок в консоли браузера (кнопка `F12`) перед отправкой на ревью. 

Напоминаю, что работа может быть принята только после исправления всех критических замечаний `Нужно исправить`.

**Комментарии `Можно лучше` не обязательны к исправлению прямо сейчас, это рекомендации**

Удачного рефакторинга кода.

Neagz committed:
Исправления согласно комментариям:
- Обернут список с ссылками тегом <nav> в navigate.hbs
- Добавлен атрибут {{name}} для инпутов в input.hbs и корректировка файлов с использованием этого шаблона
- Вынесены все цвета в отдельные переменные в style.css

mikebars1995 reviewed (commented):
Поздравляю! Ваша работа принята.

Вы отлично потрудились.

Удачного дальнейшего обучения.
